README.md

```diff
@@ -12,3 +13,6 @@ the function and run automatically when you commit through a GitHub action.
 
 What is the span of the parallel program, in terms of worst-case $\Theta$? Hint:
 It may help to consider the DAG of the parallel program.
+
+## Answer
+Since this fits the form of divide and conquer, the longest path on the DAG contains $2n+1$ nodes, so the span is $T_{\infty}(n) \in \Theta(\log n)$.
```

Comment:
Wait, $2n+1$ and therefore $\log n$? And does it really fit what we talked about for counting matches? Isn't there more work to be done here?